packages/components/inputs/search-select-input/src/search-select-input.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import {
+  SafeHTMLElement,
+  filterAriaAttributes,
+} from '@commercetools-uikit/utils';
+import SearchIconDropdownIndicator from '../../../../../src/components/internals/search-icon-dropdown-indicator';
+import AsyncSelectInput from '../../async-select-input';
```

Comment:
As you can notice that a relative import is used instead of using the package import. The reason is that there are some changes introduced into the `select-utils` and `async-select-input` components, however they are not yet published and hence the import of the current version will lack those changes. I see two ways forward.

1. Let the UX/UI review finish, which will make certain that we do not need any more changes in the underlying components. Publish this component with with relative imports. After it is published (the changes for the other components will also be published), create a new PR for this component, replacing the relative import with the package imports.
2. Let the UX/UI review finish, which will make certain that we do not need any more changes in the underlying components. Put this PR on hold. Create a new PR for the underlying components (`async-select-input and select-utils`), publish them. Then resume this PR and with the packages import and then publish this component

Reply:
I'd say you can already move forward with 2. and create a PR for the async select-input and utils 🙂

Reply:
https://github.com/commercetools/ui-kit/pull/1554
https://github.com/commercetools/ui-kit/pull/1555